profile/README.md

```diff
@@ -26,7 +25,10 @@ and platform-portable, written in Go
 * [Plugins](https://github.com/orgs/arcalot/repositories?q=arcaflow-plugin-), written in
 various languages and delivered as Linux containers or modules
 * [Workflow definitions](https://github.com/arcalot/arcaflow-workflows), written in
-YAML, tieing together plugins and their data
+YAML, tying together plugins and their data
+
+<br clear="left"/>
+<br clear="left"/>
```

Comment:
Is this supposed to be repeated?  (Does the second one need the `clear` directive?)

Reply:
This was repeated because github stripped the style spacing around the image, and you pointed out that spacing was too close. The `<br/>` is the only control I have over vertical spacing AFAIK. The clear probably isn't strictly needed in the second one.